tests/functional/delete_runs/delete_runs_test_files/project_info.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "simple",
```

Comment:
If we want to keep this please move it to the other test projects `tests/projects/` with some name like `cpp_noerror`.
With a small update we can change the current framework so it is selectable which project is used for the tests.